packages/svelte-undp-design/src/lib/Breadcrumbs/Breadcrumbs.stories.ts

```diff
@@ -25,6 +25,10 @@ const meta = {
 		breadcrumbs: {
 			description: `Breadcrumbs data to be shown. Each object should contain 'name', 'icon' and 'url'. URL is optional. Icon can use either fontawesome of any icon image's URL.`,
 			defaultValue: []
+		},
+		fontSize: {
+			description: 'Font size either large, medium or small',
+			defaultValue: 'medium'
```

Comment:
probably, you can add `type = string`, and parameters to specify choices of parameter

see the following example.

https://github.com/UNDP-Data/geohub/blob/be14dc3430a49ad3cbf5c84f016e8d217d18010f/packages/svelte-undp-design/src/lib/Button/Button.stories.ts#L10-L30